ecommerce/admin.py

```diff
@@ -0,0 +1,75 @@
+"""
+Admin views for ecommerce models
+"""
+
+from django.contrib import admin
+
+from ecommerce.models import (
+    Line,
+    Order,
+    OrderAudit,
+    Receipt,
+)
+from bootcamp.utils import get_field_names
+
+
+class LineAdmin(admin.ModelAdmin):
+    """Admin for Line"""
+    model = Line
+
+    readonly_fields = get_field_names(Line)
+
```

Comment:
Adding a list display would help. Even better if we could display the klass title

```
    list_display = ('description', 'klass_id', 'price', 'order', )
```